CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Pending changes
 
+–
+
+---
+
+## 1.0.5
+
 - [#370](https://github.com/bumble-tech/appyx/pull/370) – **Added**: `RetainedInstanceStore` has two new functions which allows a developer to check whether an object is retained. These are: `isRetainedByStoreId` and `isRetained`.
 
```

Comment:
Please add date similar to other blocks below

Reply:
done